docs/guides/adding_a_codemod.md

```diff
@@ -0,0 +1,13 @@
+# Adding a codemod
+
+If you need to add a discoverable codemod, then you need to
+
+1. figure out the correct repository<br />
+   If you want to provide a codemod for the `app-runtime` library, then the
+   library's repository is the correct location
+1. put the file into a `codemods` folder on the root level<br />
```

Comment:
```suggestion
2. put the file into a `codemods` folder at the root level of the package<br />
```

Reply:
I changed the test, I kept the `.1` as markdown automatically increments it, so it's less hassle to add items somewhere inbetween